CMS-Enterprise/mint-app
# [MINT-3259] Allow linking milestone on add common solution

# MINT-3259

## Description


This PR allows for adding milestone links when adding common solutions

## How to test this change

1. Run scripts/dev test:go and validate unit test success
2. Run Postman endpoint to add common solution and validate that milestone links are added as expected

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
